modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -18,18 +19,140 @@ As an Operator author, your Operator must meet additional requirements to run pr
 // TODO: Include more info w/ better steps on how to do this:
 //* You must understand the {product-title} proxy configuration.
 
-For the CSV requirements, you can make the following changes as the Operator author.
+For the CSV requirements, you can make the following changes as the Operator author. The following procedure uses the Memcached Operator created in the Operator SDK tutorials for Go, Ansible, and Helm.
 
 .Prerequisites
 
 * An Operator project with a CSV.
 
 .Procedure
 
-. Use SHA references to related images in two places in the CSV for your Operator:
+. Set an environment variable, such as `RELATED_IMAGE_MEMCACHED`, for any additional images used by the Operator in the `config/manager/manager.yaml` file:
++
+.Example `config/manager/manager.yaml`
+[%collapsible]
+====
+[source,yaml]
+----
+...
+spec:
+  ...
+    spec:
+      ...
+      containers:
+      - command:
+        - /manager
+        ...
+        env:
+        - name: RELATED_IMAGE_MEMCACHED
+          value: "docker.io/memcached:1.4.36-alpine"
+----
+====
+
+. Pull the related images with digests by replacing hard-coded image references with environment variables in the following files:
+
+* For Go-based Operator projects, update the `controllers/memcached_controller.go` file with the following changes:
```

Comment:
This kind of goes along with my comment above. I think maybe only one or the other should have "with the following changes" it just seems a bit confusing to me. Do users need to make the changes in each file?

It could be something as simple as adding a small addition to number 2 that discusses what type of operator project gets updated maybe? Just a suggestion:

> Pull the related images with digests by replacing hard-coded image references with environment variables for Go, Ansible, and Helm-based Operator Projects. 

And then i think you could leave the bullets as is.

Reply:
It would be better to be more direct:
```suggestion
* For Go-based Operator projects, add the environment variable to the `controllers/memcached_controller.go` file as shown in the following example:
```